### DISCUSSION TITLE
An error occurred importing the selected file with “Better BibTeX”. Please ensure that the file is valid and try again.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [retorquere/zotero-better-bibtex#2503] [General]
On Ubuntu 22.10, I try to import the following item from clipboard into Zotero with the help of BBT :

```bibtex
@misc{ GAP4.13dev,
  title =            {{GAP} {\textendash} {G}roups, {A}lgorithms, and {P}rogramming, {V}ersion 4.13dev},
  organization =     {The GAP {G}roup},
  howpublished =     {\href {https://www.gap-system.org} {\texttt{https://www.gap-system.org}}},
  key =              {GAP},
  keywords =         {groups; *; gap; manual}
}
```
But the following error was triggered:

![image](https://user-images.githubusercontent.com/11155854/235638680-bef5235a-cc54-4a10-a7a0-713abbcb9a57.png)

Any tips for achieving this goal?

Regards,
Zhao

**Comment by retorquere** [comment]
There's an error in my bibtex parser. Working on a fix.

**Reply by retorquere** [reply]
The build posted on #2504 will fix the problem. I'd appreciate it if you could verify, then I'll cut a new release.
